EasyPostTest/ClientFactoryTest.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using EasyPost;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace EasyPostTest
+{
+    [TestClass]
+    public class ClientFactoryTest {
+
+        private class FakeClientFactoryForKeyPlusUrl : IClientFactory {
```

Comment:
I think this was probably because of the 'nameof' mistake?  the csproj files appear correct, I cannot repro.